Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/usr/local/lib/python3.10/dist-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/local/lib/python3.10/dist-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.10/dist-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/usr/local/lib/python3.10/dist-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/usr/local/lib/python3.10/dist-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
scrape_cbnc(3)
------------------

----- stdout -----
done[1] > Korupsi Ba
----- stdout -----
done[2] > Beres-bere
----- stdout -----
done[3] > Tega-tegan
------------------

[0;31m[0m
[0;31mAttributeError[0mTraceback (most recent call last)
[0;32m<ipython-input-3-780341ea800f>[0m in [0;36m<cell line: 1>[0;34m()[0m
[0;32m----> 1[0;31m [0mscrape_cbnc[0m[0;34m([0m[0;36m3[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m<ipython-input-2-07d31b7729ff>[0m in [0;36mscrape_cbnc[0;34m(hal)[0m
[1;32m     17[0m                 [0mge_[0m [0;34m=[0m [0mreq[0m[0;34m.[0m[0mget[0m[0;34m([0m[0mlink_art[0m[0;34m,[0m[0mhades[0m[0;34m)[0m[0;34m.[0m[0mtext[0m[0;34m[0m[0;34m[0m[0m
[1;32m     18[0m                 [0msop_[0m [0;34m=[0m [0mbs[0m[0;34m([0m[0mge_[0m[0;34m,[0m[0;34m'lxml'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 19[0;31m                 [0mart[0m [0;34m=[0m [0msop_[0m[0;34m.[0m[0mfind[0m[0;34m([0m[0;34m'div'[0m[0;34m,[0m[0mclass_[0m[0;34m=[0m[0;34m'lm_content mt10'[0m[0;34m)[0m[0;34m.[0m[0mfind[0m[0;34m([0m[0;34m'article'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     20[0m                 [0mheadline[0m [0;34m=[0m [0mart[0m[0;34m.[0m[0mfind[0m[0;34m([0m[0;34m'h1'[0m[0;34m)[0m[0;34m.[0m[0mtext[0m[0;34m.[0m[0mstrip[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m     21[0m                 [0mwarp[0m [0;34m=[0m [0mart[0m[0;34m.[0m[0mfind_all[0m[0;34m([0m[0;34m'div'[0m[0;34m,[0m[0mclass_[0m[0;34m=[0m[0;34m'detail_text'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mAttributeError[0m: 'NoneType' object has no attribute 'find'

